src/contribute/dependencies.md

```diff
@@ -55,13 +55,14 @@ These kind of urgently required upgrades might not get automatically picked up b
 ## JIRA Automation
 
 In order to track the dependency upgrade process, JIRA tickets will be created per significant outdated dependency based on the report. A bot named *Beam Jira Bot* was created for managing JIRA issues. Beam community agrees on the following policies that creates and updates issues.
-* Issues will be named as "Beam Dependency Update Request: <dep_name> <dep_newest_version>".
-* Issues will be created under the component *"dependencies"*
-* Issues will be assigned to the primary owner of the dependencies, who are mentioned in the dependency ownership files. ([Java Dependency Owners](https://github.com/apache/beam/blob/master/ownership/JAVA_DEPENDENCY_OWNERS.yaml) and [Python Dependency Owners](https://github.com/apache/beam/blob/master/ownership/PYTHON_DEPENDENCY_OWNERS.yaml))
-* If more than one owners found for a dependency, the first owner will be picked as the primary owner, the others will be pinged in the issue's description.
-* If no owners found, leave the assignee empty. The component lead is responsible for triaging the issue.
-* Avoid creating duplicate issues. Updating the descriptions of the open issues created by the previous dependency check.
-* The dependency sometimes is not able to be upgraded, the issue should be closed as *"won't fix"*. And, the bot should avoid recreating issues with "won't fix".
+* Issues will be named as "Beam Dependency Update Request: <dep_name>".
+* Issues will be created under the component *"dependencies"*.
+* The bot will pings owners in the issue descriptions, who are listed in the dependency ownership files. ([Java Dependency Owners](https://github.com/apache/beam/blob/master/ownership/JAVA_DEPENDENCY_OWNERS.yaml) and [Python Dependency Owners](https://github.com/apache/beam/blob/master/ownership/PYTHON_DEPENDENCY_OWNERS.yaml))
+* To avoid creating duplicate issues, the bot updates the descriptions of the opening issues created that reminds owners review and upgrade a dependency.
+* The dependency sometimes is not able to be upgraded, the issue should be closed as *"won't fix"*.
+* The bot reopens the issue in two conditions:
+  * Contributors could specify a fix version on the JIRA issue that indicates when the dependency would be upgraded in the future. The bot will automatically reopen the issue once the comming Beam SDK release version hits the fix version.
+  * If an issue has been closed more than 6 months and there are 3 or more releases since closure, the bot will reopen the issue and ask owners to review the new version of the dependency.
```

Comment:
Six months **and** three or more minor releases have passed since the JIRA was closed.